build_all.bat

```diff
@@ -195,4 +195,12 @@ call group5\build.bat 2019.3.5f1
 call group5\build.bat 2019.3.6f1
 call group5\build.bat 2019.3.7f1
 call group5\build.bat 2019.3.8f1
+call group5\build.bat 2019.3.9f1
+call group5\build.bat 2019.3.18f1
```

Comment:
use sorted order - also, what happened to 3.10 and 3.15~3.17?

Reply:
😂😂😂I don't know how I managed to type 18 instead of 10 but fixed now via force push :)